xbmc/pvr/addons/PVRClient.cpp

```diff
@@ -546,7 +560,7 @@ PVR_ERROR CPVRClient::GetDriveSpace(long long& iTotal, long long& iUsed)
   return DoAddonCall(__FUNCTION__, [this, &iTotal, &iUsed](const AddonInstance* addon) {
```

Comment:
```
xbmc/pvr/addons/PVRClient.cpp:560:37: warning: lambda capture 'this' is not used [-Wunused-lambda-capture]
  return DoAddonCall(__FUNCTION__, [this, &iTotal, &iUsed](const AddonInstance* addon) {
                                    ^~~~~
1 warning generated.
```

Reply:
Was it only be there or also on other places, e.g. `GetChannelGroups` ...?
Has you becomed them from Jenkins or from another tester, if yes have you a link to it?

Thanks for report about.

Reply:
This is the only place.
https://jenkins.kodi.tv/job/Android-ARM/17742/clang/new/

Reply:
Have looked in jenkins that a good feature to see:
- GNU C Compiler (gcc) Warnings
- Clang (LLVM based) Warnings

Reply:
This existed since August 2017, also did fail builds if new warnings got intoduced. So one could have a look at the code and fix it.
This was removed in March, for whatever reason.